packages/hooks/src/useContentFlag.ts

```diff
@@ -0,0 +1,18 @@
+import { ContentFlag } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+
+const useContentFlag = () => {
```

Comment:
@sergei-deriv Would you help to write tests for it 🙏🏻 You can check the test files in the hooks package for reference 🙇🏻